internal/pkg/cmd/prerunner.go

```diff
@@ -448,6 +448,9 @@ func (r *PreRun) AuthenticatedWithMDS(command *AuthenticatedCLICommand) func(*co
 
 		// Even if there was an error while setting the context, notify the user about any unmet run requirements first.
 		if err := ErrIfMissingRunRequirement(cmd, r.Config); err != nil {
+			if err == v1.RunningOnPremCommandInCloudErr && strings.Contains(cmd.CommandPath(), "confluent cluster") {
+				return v1.RunningClusterCommandInCloudErr
```

Comment:
Why is this variable global/public? It can probably just be returned here without being defined.

Reply:
similar errors are all defined in config.go so i want to put them together

Reply:
The errors in that file are related to being logged in to the wrong context, but this specific suggestion is about a potential typo that the user may have made while typing. Those should be separate, IMO.

Is there a way we can take the existing error and append a suggestion to it so we can avoid duplication? I noticed you copied and pasted this message "Log in to Confluent Platform with `confluent login --url <mds-url>` to use `confluent cluster`." which makes it more work to maintain in the future.